src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -897,5380 +717,1211 @@ public void Dispose(bool disposing)
-    }
-
-    if ($ret -eq $true)
-    {
-        $result[$computerName] = $true
-    }
-    else
-    {
-        $result[$computerName] = $false
-    }
-}
-$result
-";
-
-        /// <summary>
-        /// The indicator to use when show progress
-        /// </summary>
-        private string[] _indicator = { "|", "/", "-", "\\" };
-
-        /// <summary>
-        /// The activity id
-        /// </summary>
-        private int _activityId;
-
-        /// <summary>
-        /// After call 'Shutdown' on the target computer, wait a few
-        /// seconds for the restart to begin.
-        /// </summary>
-        private const int SecondsToWaitForRestartToBegin = 25;
-
-        /// <summary>
-        /// Actual time out in seconds
-        /// </summary>
-        private int _timeoutInMilliseconds;
-
-        /// <summary>
-        /// Indicate to exit
-        /// </summary>
-        private bool _exit, _timeUp;
-        private readonly CancellationTokenSource _cancel = new CancellationTokenSource();
-
-        /// <summary>
-        /// A waithandler to wait on. Current thread will wait on it during the delay interval.
-        /// </summary>
-        private readonly ManualResetEventSlim _waitHandler = new ManualResetEventSlim(false);
-        private readonly Dictionary<string, ComputerInfo> _computerInfos = new Dictionary<string, ComputerInfo>(StringComparer.OrdinalIgnoreCase);
-
-        // CLR 4.0 Port note - use https://msdn.microsoft.com/en-us/library/system.net.networkinformation.ipglobalproperties.hostname(v=vs.110).aspx
-        private readonly string _shortLocalMachineName = Dns.GetHostName();
-
-        // And for this, use PsUtils.GetHostname()
-        private readonly string _fullLocalMachineName = Dns.GetHostEntryAsync("").Result.HostName;
-
-        private int _percent;
-        private string _status;
-        private string _activity;
-        private Timer _timer;
-        private System.Management.Automation.PowerShell _powershell;
-
-        private const string StageVerification = "VerifyStage";
-        private const string WmiConnectionTest = "WMI";
-        private const string WinrmConnectionTest = "WinRM";
-        private const string PowerShellConnectionTest = "PowerShell";
-
-#endregion "parameters and PrivateData"
-
-#region "IDisposable Members"
-
-        /// <summary>
-        /// Dispose Method
-        /// </summary>
-        public void Dispose()
-        {
-            this.Dispose(true);
-            // Use SuppressFinalize in case a subclass
-            // of this type implements a finalizer.
-            GC.SuppressFinalize(this);
-        }
-
-        /// <summary>
-        /// Dispose Method.
-        /// </summary>
-        /// <param name="disposing"></param>
-        public void Dispose(bool disposing)
-        {
-            if (d
```

Comment:
Define `5` as a const int with a meaningful name.  Or perhaps 0,1,5 should be an enum.

Reply:
I'll leave it as an int as that's really what it has to be turned into later (see https://msdn.microsoft.com/en-us/library/aa394058(v=vs.85).aspx)